examples/plot_map_editcolormap.py

```diff
@@ -0,0 +1,36 @@
+"""
+==============================
+Editing Map Colormaps and Norm
+==============================
+
+A simple example to show how to edit the
+display of a map
+"""
+
+from sunpy.data.sample import AIA_171_IMAGE
+import sunpy.map
+import matplotlib.cm as cm
+import matplotlib.colors as colors
+import matplotlib.pyplot as plt
+
+###############################################################################
+# We now create the Map using the sample data.
+
+aiamap = sunpy.map.Map(AIA_171_IMAGE)
+
+###############################################################################
+# Now lets replace the colormap which sets the colors as well as the
+# normalization which sets how data values are translated to colors
+
+aiamap.plot_settings['cmap'] = cm.Greys_r
```

Comment:
does this have to be a colourmap object or can it be a string? A string would be easier.